public/Electron/ipc-handler.js

```diff
@@ -16,66 +14,86 @@ const registerIpcHandlers = () => {
         properties: ["openDirectory"],
       });
 
-      if (!canceled && filePaths.length > 0) {
-        [fileInfo.filePath] = filePaths;
+      if (canceled) return new Error("open dialog failed");
 
-        const reactreePath = path.join(userHomeDir);
-        exec(
-          `ln -s ${reactreePath}/Desktop/reactree-frontend/src/utils/reactree.js ${filePaths}/src/Symlink.js`,
-          (error, stdout, stderr) => {
-            handleErrorMessage(stderr);
-          },
-        );
+      const filePath = filePaths[0];
+      const userHomeDir = os.homedir();
+      const homeDirectory = path.join(userHomeDir);
```

Comment:
`path.join` 의 인자로 경로를 1개만 넣는다면 사용하지 않는 것과 같을 것 같아요
```suggestion
      const userHomeDir = os.homedir();
```